install/kubernetes/helm/istio/charts/pilot/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
What is this ? Why in pilot ?

Reply:
This is the standard way to install CRs. Helm applies YAMLs in an arbitrary order, so you have to do round-about way to force ordering with hooks.

Reply:
Did you test with helm template as well ( i.e. no helm ) ?

Reply:
No, I don't have a way to run locally. CI/CD passed the tests.